data/transactions/logic/eval.go

```diff
@@ -211,7 +211,6 @@ type LedgerForLogic interface {
 	AssetParams(aidx basics.AssetIndex) (basics.AssetParams, basics.Address, error)
 	AppParams(aidx basics.AppIndex) (basics.AppParams, basics.Address, error)
 	OptedIn(addr basics.Address, appIdx basics.AppIndex) (bool, error)
-	GetCreatableID(groupIdx int) basics.CreatableIndex
```

Comment:
We no longer track creatables in the ledger, since ledgers are shared for inner transactions, but "racked creatables" used only the transaction index for tracking.